src/Playwright/Transport/Connection.cs

```diff
@@ -262,7 +261,7 @@ private void Transport_MessageReceived(object sender, MessageReceivedEventArgs e
             }
             catch (Exception ex)
             {
-                Close(ex);
+                DoClose(ex);
```

Comment:
normally I'd call it just Close, but since there is an event already, we had to choose a different name.